src/http/headers.cr

```diff
@@ -275,7 +285,18 @@ struct HTTP::Headers
 
   forward_missing_to @hash
 
+  def serialize(io)
+    @hash.each do |key, values|
+      cast(values).each do |value|
+        io << key.name << ": " << value << "\r\n"
+      end
+    end
```

Comment:
Why don't you re-use `#each` here? The implementation is identical.

If you want to improve performance, a better solution would do without calling `cast` to avoid array allocation.

Reply:
I'm using `#each` now